gatling-recorder/src/main/scala/io/gatling/recorder/scenario/template/RequestTemplate.scala

```diff
@@ -51,10 +51,12 @@ private[scenario] object RequestTemplate {
         }.getOrElse("")
 
       def renderLongString(value: String) =
-        if (value.nonEmpty)
-          value.grouped(65535).map(protectWithTripleQuotes).mkFastring(" + ")
-        else
-          protectWithTripleQuotes("")
+        if (value.size > MaxLiteralSize) {
+          wrapLongString(value.grouped(MaxLiteralSize).map(protectWithTripleQuotes).mkFastring(", "))
```

Comment:
Haven't find mkFastring implementation with prefix and suffix like, replaced with fast"""Seq(${value.grouped(MaxLiteralSize).map(protectWithTripleQuotes).mkFastring(", ")}).mkString"""